base/functions/Dockerfile

```diff
@@ -1,2 +1,2 @@
-FROM functions
+FROM functions-node-14
```

Comment:
This is gonna update node in all functions.

The idea was to gradually introduce node 14 in each function by changing its [Dockerfile](https://github.com/pagopa/io-mock/blob/df6deafe4e3bb05d828b7762b2628080b635d9e8/io-functions-eucovidcerts/Dockerfile#L1) and the [docker-compose.yml](https://github.com/pagopa/io-mock/blob/df6deafe4e3bb05d828b7762b2628080b635d9e8/docker-compose.yml#L263)

Reply:
This is breaking the docker compose override. As far as I know functions should all be migrated to node 14. Can you spot which remained on node 10?

Reply:
>  As far as I know functions should all be migrated to node 14

True, the idea was to move all of them to function-node-14 and then remove function.